compose/cli/main.py

```diff
@@ -216,7 +216,7 @@ class TopLevelCommand(object):
       build              Build or rebuild services
       config             Validate and view the Compose file
       create             Create services
-      down               Stop and remove containers, networks, images, and volumes
+      down               Stop and remove containers, networks, images, and volumes (option -v)
```

Comment:
Perhaps

```suggestion
      down               Stop and remove containers, networks, images and, optionally, volumes
```

Or the description should be more "generic"; don't list all object types, but something like;

```
      down               Stop and remove resources
```

(perhaps a slightly longer description than that)

Reply:
Makes sense. I will change.

Reply:
(I have two users, one of them has already key for signing, this is why seems like two people are making this)

Reply:
I will change to the more generic

Reply:
Fixed, @thaJeztah